File: Java/Booking/CreateBasketResult.java
package LTD_API;

public class CreateBasketResult
{
    private boolean success;
    private String basketId;
    
    public boolean getSuccess()
    {
        return success;
    }
    public void setSuccess(boolean success)
    {
        this.success = success;
    }
    
    public String getBasketId()
    {
        return basketId;
    }
    public void setBasketId(String basketId)
    {
        this.basketId = basketId;
    }
}

File: PHP/Booking/CreateBasketResult.php
<?php
namespace LTD_API;

class CreateBasketResult
{
    private $success;
    private $basketId;

    public function getSuccess()
    {
        return $this->success;
    }
    public function setSuccess($success)
    {
        $this->success = $success;
    }

    public function getBasketId()
    {
        return $this->basketId;
    }
    public function setBasketId($basketId)
    {
        $this->basketId = $basketId;
    }
}

